js/components/urlBar.js

```diff
@@ -368,6 +369,14 @@ class UrlBar extends ImmutableComponent {
       action='#'
       id='urlbar'
       ref='urlbar'>
+      <Button iconClass={this.titleMode ? 'fa-star' : 'fa-star-o'}
```

Comment:
It is, It should be on another branch, not sure how it slipped in. This change is to tackle the new bookmark design.